app/models/participant_declaration.rb

```diff
@@ -1,68 +1,71 @@
 # frozen_string_literal: true
 
 class ParticipantDeclaration < ApplicationRecord
-  has_many :profile_declarations
-  has_one :current_profile_declaration, -> { order(created_at: :desc) }, class_name: "ProfileDeclaration"
-  has_one :participant_profile, through: :current_profile_declaration
+  has_one :profile_declaration
```

Comment:
We will need to cut out the old ones (if there are any) in sandbox and prod.

Reply:
There weren't when I last checked, but yes. There should only be a maximum of one when this is deployed.